template/.stylelintrc

```diff
@@ -1,9 +1,117 @@
 {
-  "extends": "stylelint-config-recommended",
+  "extends": ["stylelint-config-recommended", "stylelint-config-idiomatic-order"],
```

Comment:
So, this syntax is no different than what we are doing with esLint. https://github.com/AlaskaAirlines/WC-Generator/blob/dsande/generator%2362/eslint/template/.eslintrc

It would be worth looking into seeing if we can add this config to the https://github.com/AlaskaAirlines/eslint-config-auro repo.

Reply:
I am only suggesting that this is a direction we should look at. But this PR is solid as is as well as https://github.com/AlaskaAirlines/WC-Generator/pull/100

Let's merge these ideas and create new stories for how to consolidate if that is an opportunity.

Reply:
I really want to be careful of adding scope to two stories at the same time when that direction is not needed to deliver these individual tickets.